tests/rules/no-dispatch-in-effects.test.ts

```diff
@@ -164,7 +156,7 @@ ruleTester().run(path.parse(__filename).name, rule, {
                 return somethingElse()
               }),
             ),
-          { dispatch: false, useEffectsErrorHandler: false },
+          { dispatch: true, useEffectsErrorHandler: false, ...options },
```

Comment:
Note that it's correct to `suggest` here, because although it doesn't have `dispatch: fase` **explicitly**, it may have **implicitly** through `options`.

Reply:
Thanks! Just for a future reference, I find these comments useful to have in the code.

```suggestion
          // Note that it's correct to suggest, instead of fix, here, because although it doesn't have dispatch: fase explicitly, it may have implicitly through options
          { dispatch: true, useEffectsErrorHandler: false, ...options },
```

Reply:
There's a typo. "fase", instead of "false" :sweat_smile:

And in that case, I prefer to put this comment above the "suggestions" key.

Reply:
Ha! 💯
I fixed it locally, and should be pushed now. 😅